### DISCUSSION TITLE
Exposing the indi allsky to internet

### DISCUSSION THREAD

**Discussion by livium2000** [aaronwmorris/indi-allsky#1616] [General]
Quick question: How can I expose my indi all sky server to the internet? Let's say I want to be available via a https domain (like noip) or wordpress installation maybe? Don't find nothing in the wiki about that. Thanks

**Comment by aaronwmorris** [comment]
If you can setup a port forward with your home router and use a dynamic DNS service, it should be relatively easy to have it exposed to the internet.

If you can setup those, you could use something like letsencrypt to setup a proper SSL certificate for the system, as well.

**Reply by livium2000** [reply]
Thanks. Will try this for sure.
